modules/ROOT/pages/getting-started.adoc

```diff
@@ -23,4 +23,8 @@ Learn how to install {productname} via {cloudname}, package managers, self-hoste
 // Empty cell to even out rows
 // | 
 
-|===
\ No newline at end of file
+|===
+
+include::partial$misc/admon-getting-started-with-tinymce.adoc[]
```

Comment:
This looks odd because there are two headings about getting started on this page

Reply:
Agree, but it was requested.